UnitTests/Helpers/Test_StorageFileHelper.cs

```diff
@@ -239,7 +239,7 @@ public void Test_StorageFileHelper_IsFilePathValid_WithCorrectFilePath()
         [TestMethod]
         public void Test_StorageFileHelper_IsFilePathValid_WithIllegalCharacters()
         {
-            string filepath = "my_folder/my_file.txt";
+            string filepath = "my>folder|my<file\t.txt";
```

Comment:
Honestly, given the size of the PR, I'd say you'd be better off and quicker just redoing it on a new branch and close this PR.